### DISCUSSION TITLE
'furthest page read' function

### DISCUSSION THREAD

**Discussion by hrrrps** [spacecowboy/Feeder#359] [Ideas]
First of, thanks for Feeder, I enjoy it very much. I want to suggest a function, or if it already exists, some instructions for enabling it. I'm not sure what the technical term for this might be, but what I'm thinking of is something like a 'furthest page read' function so that if you stop reading an article you can come back to it and continue reading from where you left off. This would be especially helpful for the longer form articles that are more difficult to finish in one sitting.

**Comment by spacecowboy** [comment]
Sounds like a great idea. Not sure how it would be implemented, but would be nice to have.
